test/expected/timestamp.out

```diff
@@ -524,6 +524,21 @@ FROM unnest(ARRAY[
   110 |         110
 (4 rows)
 
+SELECT time, time_bucket(10, time)
+FROM unnest(ARRAY[
+     '-1',
+     '0',
+     '1',
+     '11'
+    ]::int[]) AS time;
+ time | time_bucket 
+------+-------------
+   -1 |         -10
```

Comment:
I'd like to see 10 and -10 as time values in this test (given my comment above).